src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
* Kindly add tests as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.) 
> Create unit tests for pure functions in your Redux code.
> Create integration tests for your application using the React Testing Library.
> You may need to mock the access to the API, so that your tests don't send actual requests.
> You may need to mock the connection to the Redux Store.

Reply:
Thanks.
I have done the test